Source/Core/Core/HW/EXI/EXI_DeviceIPL.cpp

```diff
@@ -98,31 +100,90 @@ void CEXIIPL::Descrambler(u8* data, u32 size)
   }
 }
 
+void CEXIIPL::Patcher(u8* data, const u32 rom_crc32)
+{
+  if (Config::Get(Config::MAIN_RAM_OVERRIDE_ENABLE))
+  {
+    // Replace SIMM fields containing the upper 16 bits of the Physical MEM1 Size (lis
+    // instructions).
+    const s16 simm_p_mem1_size = Common::swap16(Memory::GetRamSizeReal() >> 16);
+    switch (rom_crc32)
+    {
+    case CRC32_NTSC_v1_0:
+    case CRC32_NTSC_v1_1:
+    case CRC32_PAL_v1_0:
+    case CRC32_MPAL_v1_1:
+      std::memcpy(data + 0x820 + 0x494 + 2, &simm_p_mem1_size, sizeof(simm_p_mem1_size));
+      std::memcpy(data + 0x100 + 0x31C + 2, &simm_p_mem1_size, sizeof(simm_p_mem1_size));
+      std::memcpy(data + 0x100 + 0x36C + 2, &simm_p_mem1_size, sizeof(simm_p_mem1_size));
+      break;
+    case CRC32_NTSC_v1_2:
+    case CRC32_PAL_v1_2:
+      std::memcpy(data + 0x820 + 0x490 + 2, &simm_p_mem1_size, sizeof(simm_p_mem1_size));
+      std::memcpy(data + 0x100 + 0x31C + 2, &simm_p_mem1_size, sizeof(simm_p_mem1_size));
+      std::memcpy(data + 0x100 + 0x36C + 2, &simm_p_mem1_size, sizeof(simm_p_mem1_size));
+      break;
+    default:
+      PanicAlertFmtT(
+          "Emulated Memory Size Override is not supported for unrecognized IPL dumps.\n");
```

Comment:
Does this intentionally add the newline at the end? There's nothing behind it, and the dialog ends; so it is either a left-over or you forgot to add something _else_ there.

Reply:
Oh right.  This is a shortened copy of a message I had in a previous version of this PR.  There used to be a second line of dialogue, yes.  I'll fix that.